docs/user/intro.rst

```diff
@@ -1,263 +0,0 @@
-.. _introduction:
-
-Introduction
-============
-
-.. toctree::
-   :maxdepth: 2
-
-
-The framework is designed for extreme modularity. Don't panic when looking at the source code.
-Read the following descriptions of the *pillars* of the framework first.
-
-
-Overview
---------
-
-A QCoDeS experiment typically consists of a Loop_ that sweeps over one or more Parameters_ of one or more Instruments_,
-measures other Parameters_ at each sweep point, and stores all of the results into a DataSet_.
-
-.. _Parameters: Parameter_
-.. _Instruments: Instrument_
-
-While the simple case is quite straightforward, it is possible to create a very general experiment by defining richer Parameters and
-by performing additional Loop actions at each sweep point.
-The overview on this page provides a high-level picture of the general capabilities;
-consult the detailed API references and the samples to see some of the complex procedures that can be described and run.
-
-Instrument
```

Comment:
i wonder if instead of removing we can move the instrument section (with some sanitization) into for example https://qcodes.github.io/Qcodes/examples/writing_drivers/Creating-Instrument-Drivers.html ? or create a completely new notebbok that talks about instruments in general and put this info there?

Reply:
Like parameters, we should have an examples/instrument notebook and consolidate this and other information about instruments there.

Reply:
then let's do that in this PR?

Reply:
See below, yes.